docs/src/main/asciidoc/sqs.adoc

```diff
@@ -636,6 +636,8 @@ Note that if there are messages available the call may return earlier than this
 - `messageVisibilitySeconds` - Set the minimum visibility for the messages retrieved in a poll.
 Note that for `FIFO` single message listener methods, this visibility is applied to the whole batch before each message is sent to the listener.
 See <<FIFO Support>> for more information.
+- `acknowledgementMode` - Set the acknowledgement mode for the container. Default value is `SqsListenerAcknowledgementMode.DEFAULT`,
```

Comment:
I think the behavior we're after is - if any `AcknowledgementMode` is set in the annotation, it takes precedence over the one set in the factory / `ContainerOptions`

Reply:
well, it's kind of stated above (line 628).

I'm down to change it anyway. What do you think of

```suggestion
- `acknowledgementMode` - Set the acknowledgement mode for the container. If any value is set, it will take precedence over the acknowledgement mode defined for the container factory options. See <<Acknowledgement Mode>> for more information.
```

?

Reply:
Looks great, thanks!